### ISSUE TITLE
ov50 - Cyclones db takes awhile to load into Climatology when the files are fully uncompressed.

### ISSUE THREAD

**Issue by rgleason** [seandepagnier/climatology_pi#54]
See https://github.com/seandepagnier/climatology_pi/pull/41#issuecomment-467078465

**Comment by rgleason** [comment]
When the data file has been removed, there is a prompt to download, when the db files are downloaded, they appear to land in the right directory, and they are all 

        _____*____.gz  compressed files.

The plugin does not read these files well. It gets through current01.gz then hangs on current02.gz.

![screenshot 71](https://user-images.githubusercontent.com/2363959/53378805-fdc2de80-3934-11e9-80f4-3b5e5ff8028b.png)

When I close the plugin and copy "current02" to the data directory and restart the plugin the plugin loads

     current.01.gz, current02, current03.gz and then hangs up on current04.gz!!

It is not the progress bar that is the problem!  I believe it is something else and the wxW progress bar is probably ok

![screenshot 72](https://user-images.githubusercontent.com/2363959/53378862-23e87e80-3935-11e9-9ad4-4fab28147fd0.png)

.Note that when I use  only the FULLY expanded files that I downloaded in the DATA directory, the plugin does load all files, however the loading of the cyclone file is much slower than normal.

![screenshot 74](https://user-images.githubusercontent.com/2363959/53378887-3c589900-3935-11e9-8875-abf53fbb17b4.png)


My question remains, isn't Climatology supposed to be able to read  *.gz files properly?
